frontend/app/src/components/dashboard/BalanceLocationIcon.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <fragment v-if="isExchange">
+    <v-img
+      contain
+      height="24"
+      :title="name"
+      :src="require(`@/assets/images/${name}.png`)"
+    />
+  </fragment>
+  <fragment v-else>
```

Comment:
Out of curiosity, didn't putting `v-if`, `v-else` on the `<v-img>` and `<v-icon/>` respectively work? From my understanding fragments are useful if you need 2 items in the root that are rendered at the same time.

Reply:
Shouldn't `v-if`/`v-else` be used on the same component type? I'll try 👍

Reply:
From what I am aware of there is no such limitation. I always used it with different types of components to be honest.